src/Collections-Strings/Symbol.class.st

```diff
@@ -410,12 +410,25 @@ Symbol >> = aSymbol [
 	^ super = aSymbol
 ]
 
+{ #category : #variables }
+Symbol >> => aVariable [
+	"If the slot we give as argument is not present in the image, we will get a nil. In that case we should throw an explicit error to the user saying a slot is missing."
+
+	aVariable ifNil: [ SlotNotFound signalForName: self ].
+	^ aVariable named: self
+]
```

Comment:
Why is this in `Symbol`? this should be at least in an extension method.

This is not good packaging...